src/app/upload/shared/audio-player.component.ts

```diff
@@ -0,0 +1,123 @@
+import { Component, Input, OnDestroy, ViewChild, ChangeDetectorRef } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { PlayerService, PlaybackMetadata } from '../../core';
+import { AudioFileModel } from '../../shared';
+
+@Component({
+  selector: 'publish-audio-player',
+  styleUrls: ['audio-player.component.css'],
+  template: `
+    <p *ngIf="error" class="error">{{error}}</p>
+
+    <button *ngIf="!playing && !loading" class="play" (click)="play()"></button>
+
+    <div #scrubber *ngIf="playing" class="scrubber"
+      (mousedown)="scrub('down', $event)"
+      (mouseup)="scrub('up', $event)"
+      (mousemove)="scrub('move', $event)">
+      <div class="meter" [style.width.%]="progress * 100"></div>
+      <div *ngIf="!dragging" class="position" [style.left.%]="progress * 100"></div>
+    </div>
+    <button *ngIf="playing" class="pause" (click)="stop()"></button>
```

Comment:
If the progess can be saved on the other file without a ton of work, pausing would be nice, but maybe a "nice to have" but not necessary. Restarting the other file seems like it would be fine too.